README.md

```diff
@@ -1,7 +1,221 @@
+- [**CSS**](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS3)
+    - The project uses **CSS** to apply the custom styles created with **SCSS** to my app. The base.html file is linked directly to the main.min.css stylesheet.
+- [**Bootstrap**](https://getbootstrap.com/)
+    - The project uses the **Bootstrap** framework to add a responsive grid system, prebuilt components, plugins built on jQuery, and Bootstrap styles to my app, before adding my custom styles.
+- [**jQuery**](https://jquery.com)
+    - The project uses **jQuery** as the primary JavaScript functionality. This is both the standard jQuery that is built with Materialize components, and my custom jQuery used in my script.js file.
+- [**Python**](https://www.python.org/)
+    - The project uses **Python** as the back-end programming language for my app.
+- [**PyMongo**](https://api.mongodb.com/python/current/)
+    - The project uses **PyMongo** as the Python API for MongoDB. This API enables linking the data from the back-end database to the front-end app.
+- [**Flask**](https://flask.palletsprojects.com/en/1.0.x/)
+    - The project uses **Flask**, which is a Python microframework.
+- [**Jinja**](https://jinja.palletsprojects.com/en/2.10.x/)
+    - The project uses **Jinja** for templating with Flask in the HTML code. I used **Jinja** to simplify my HTML code, avoid repetition, and allow simpler linking of the back-end to the front-end.
+- [**MongoDB**](https://cloud.mongodb.com/)
+    - The project uses **MongoDB** to store the database in the cloud. The information displayed in the front-end app is pulled from the database store.
+- [**Google Fonts**](https://fonts.google.com/)
+    - The project uses **Google Fonts** to style the text and suit my chosen theme. The brand logo uses the *_Dancing Script_* font and the rest of the site uses the *_Roboto_* font.
+- [**Font Awesome**](https://fontawesome.com/)
+    - The project uses **Font Awesome** for the various icons in my app.
+- [**Gitpod**](https://www.gitpod.io/)
+    - I've used **Gitpod** as the development environment to write the code for my website.
+
+# **5. Mockups
+
+# **6. Version Control
+
+- [**Git**](https://git-scm.com/)
+    - I've used **Git** as a version control system to regularly add and commit changes made to project in Gitpod, before pushing them to GitHub.
+- [**GitHub**](https://github.com/)
+    - I've used **GitHub** as a remote repository to push and store the committed changes to my project from Git.
+
+# **6.1 Hosting
+
+- [**Heroku**](https://www.heroku.com/)
+    - I've used **Heroku** as the hosting platform to deploy my app.
+
+# **7. TESTING**
+
+I used Google Chrome's Development tools to constantly test each change that I<br/> 
+made to my project and to ensure that it appeared in the desired way on different screen sizes. <br/>
+I also tested my app on different screen sizes (mobile, tablet and desktop) to ensure <br/>
+it appeared in the desired way on different devices.
```

Comment:
Also write automated tests and include the command to run if one what's to run your automated test